js/extensions/cadastrapp/components/PlotSelection.jsx

```diff
@@ -0,0 +1,200 @@
+            defaultActiveKey={props.active}>
+            <Row className="clearfix">
+                <Col sm={12}>
+                    <Nav bsStyle="tabs">
+                        {props.data.map((value, index) => (
+                            <NavItem eventKey={index}>
+                                {"Selection " + (index + 1).toString()}
+                            </NavItem>
+                        ))}
+                        <PlotSelectionTabActionButtons {...props}/>
+                    </Nav>
+                </Col>
+                <PlotSelectionTabContent {...props}/>
+            </Row>
+        </Tab.Container>
+    );
+}
+
+function PlotSelectionTabsWithDropdown(props) {
+    let otherSelectionIndex = props.active + 1;
+    if (otherSelectionIndex < 3) {
+        otherSelectionIndex = "3+ ";
+    }
+    return (
+        <Tab.Container
+            onSelect={props.onTabChange}
+            activeKey={props.active}
+            defaultActiveKey={props.active}>
+            <Row className="clearfix">
+                <Col sm={12}>
+                    <Nav bsStyle="tabs">
+                        <NavItem eventKey={0}>
+                            Selection 1
+                        </NavItem>
+                        <NavItem eventKey={1}>
+                            Selection 2
+                        </NavItem>
+                        <NavDropdown title={"Selection " + otherSelectionIndex}>
+                            {props.data.slice(2).map((value, index) => (
+                                <MenuItem eventKey={index + 2}>
+                                    {"Selection " + (index + 2 + 1).toString()}
+                                </MenuItem>
+                            ))}
+                        </NavDropdown>
+                        <PlotSelectionTabActionButtons {...props}/>
+                    </Nav>
+                </Col>
+                <PlotSelectionTabContent {...props}/>
+            </Row>
+        </Tab.Container>
+    );
+}
+
+function PlotSelectionTopActionButtons(props) {
+
+
+    let isAtleastOneSelected = false;
+    if (props.data.length > 0) {
+        let data = props.data[props.active];
+        for (let i = 0; i < data.length; i++) {
+            if (data[i][5]) {
+                isAtleastOneSelected = true;
+                break;
+            }
+        }
+    }
+
+    return (
+        <ButtonGroup className="pull-right">
+            <OverlayTrigger placement="bottom" overlay={<Tooltip>{"Zoom"}</Tooltip>}>
+                <Button
+                    onClick={() => { props.onClick("zoom"); }}>
+                    <Glyphicon glyph="zoom-in" />
+                </Button>
+            </OverlayTrigger>
+            <OverlayTrigger placement="bottom" overlay={<Tooltip>{"Landry Information"}</Tooltip>}>
+                <Button
+                    {...(!isAtleastOneSelected ? { disabled: 'true' } : {})}
+                    onClick={() => { props.onClick("landry"); }}>
```

Comment:
Can you take https://github.com/georchestra/mapstore2-cadastrapp/issues/12#issuecomment-704301264 into account for those two occurences ? thanks !